src/core/simulation.jl

```diff
@@ -250,77 +257,63 @@ they passed to the original Simulation.
 
 # Arguments
 - `directory::AbstractString`: the directory returned from the call to serialize
-- `stage_info::Dict`: Two-level dictionary containing stage parameters that cannot be
-  serialized. The outer dict should be keyed by the stage name. The inner dict must contain
+- `problem_info::Dict`: Two-level dictionary containing problem parameters that cannot be
```

Comment:
Should this be a struct?

Reply:
This is for de-serialization which is currently broken.

Reply:
I think this can remain a dict